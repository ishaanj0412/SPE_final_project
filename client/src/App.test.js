import { render, screen } from '@testing-library/react';
import * as renderer from "react-test-renderer";
import App from './App';

it("Renders the App correctly", () => {
  const tree = renderer
    .create(<App />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
