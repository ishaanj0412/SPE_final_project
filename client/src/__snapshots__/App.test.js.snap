// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the App correctly 1`] = `
<div
  className="App"
/>
`;
